liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/DevMojo.java

```diff
@@ -864,7 +882,17 @@ public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs)
 
     }
     
-
-
-
+    private boolean warXmlExists(File dir){
+        boolean looseApp = false;
+        if (dir.exists() && dir.isDirectory()) {
+            File[] files = dir.listFiles();
+            for (File file : files) {
+                if (file.isFile() && file.getName().endsWith(".war.xml")) {
```

Comment:
We should also check if the xml file is the current app.  i.e. check if the filename starts with `DevMojo.this.project.getArtifactId()`